libraries/AP_Common/AP_Time.h

```diff
@@ -2,10 +2,10 @@
  * AP_Time.h
  */
 
-#ifndef AP_TIME_H
-#define AP_TIME_H
+#pragma once
 
 #include <AP_Common/AP_Common.h>
+#include <time.h>
```

Comment:
I see that @khancyr suggested this change, but I don't see why. We should put includes in cpp files as much as we can.